packages/eslint-plugin/src/index.ts

```diff
@@ -1,4 +1,3 @@
-export type * from "./eslint-define-config";
```

Comment:
Ah I see... you should not do this, instead write an `"exports"` block in your package.json so that customers can use `/// <reference types="@eslint-react/eslint-plugin/eslint-define-config" />`

```jsonc
{
  // something like this
  "exports": {
    "./eslint-define-config.d.ts": {
      "types": "./src/index.ts"
    }
  }
}
```